tables/bg.utb

```diff
@@ -0,0 +1,196 @@
+# Bulgarian 6 dots Literary Braille
+#
+#  Copyright (C) 2021 Румяна Каменска <rkamenska@gmail.com>
+#
+#  This file is part of liblouis.
+#
+#  liblouis is free software: you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as
+#  published by the Free Software Foundation, either version 2.1 of the
+#  License, or (at your option) any later version.
+#
+#  liblouis is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with liblouis. If not, see
+#  <http://www.gnu.org/licenses/>.
+
+#-index-name: Bulgarian, uncontracted
+#-display-name: Bulgarian braille
+#
+#+locale: bg
+#+type: literary
+#+dots: 6
+#+contraction: no
+
+# Created by Rumyana Kamenska <rkamenska@gmail.com>
+
+# It includes Bulgarian, Russian, English letters and additional letters for Esperanto
+
+include spaces.uti
+include bg-6dots.dis
```

Comment:
Since we're in the process of separating display tables from translation tables, I think we shouldn't accept new translation tables that include a display table.

Reply:
I removed the import